src/Client/Pages/FilePicker/FilePickerView.fs

```diff
@@ -257,20 +257,23 @@ type FilePicker =
 
 
     static member Main(model: Model, dispatch, containerQueryClass: string) =
+        let hasFiles = not (List.isEmpty model.FilePickerState.FileNames)
         Html.div [
             prop.className "swt:flex swt:flex-col swt:gap-2 swt:overflow-y-hidden"
             prop.children [
-                if model.FilePickerState.FileNames.Length > 0 then
+                FilePicker.UploadButtons(model, dispatch, containerQueryClass)
+                if hasFiles then
                     FilePicker.FileSortElements model dispatch
                 Html.div [
                     prop.className "swt:overflow-y-auto swt:overflow-x-hidden swt:py-2"
                     prop.children [
-                        match model.FilePickerState.FileNames with
-                        | [] -> FilePicker.UploadButtons(model, dispatch, containerQueryClass)
-                        | _ -> FilePicker.FileViewTable model dispatch
+                        if hasFiles then
```

Comment:
Can propably remove this and just move this + the action buttons below to the `if hasFiles then` case in line 265